src/LazyOperations/SymmetricIntervalHull.jl

```diff
@@ -112,6 +112,24 @@ function radius_hyperrectangle(sih::SymmetricIntervalHull{N}) where {N<:Real}
     return sih.cache
 end
 
+"""
+    center(sih::SymmetricIntervalHull{N}, i::Int) where {N<:Real}
+
+Return the center along a given dimension of a symmetric interval hull of a convex set.
+
+### Input
+
+- `sih` -- symmetric interval hull of a convex set
+- `i` -- dimension of interest
+
+### Output
+
+The center along a given dimension of the symmetric interval hull of a convex set.
+"""
+function center(sih::SymmetricIntervalHull{N}, i::Int) where {N<:Real}
+    return zero(N)
```

Comment:
This function, and also the one below for `ZeroSet`, would be returning `zero(N)` even if the index `i` is out of bounds ie. if it's greater than the dimension of the set. Though a user error, it doesn't harm to actually check that the index is correct and return and error otherwise. It can be fixed with a negligible tax in runtime using `@boundscheck` macro, see https://docs.julialang.org/en/v1/devdocs/boundscheck/
As explained in that docs, `@inbounds` will normally elide this check one layer further, eg. if `center(H, i)` is used in a loop that was annotated with `@inbounds`.

Then the implementation would be (note the `@inline` macro, i would suggest that we use it for the new function `center(X, i)` in other places of this PR as well):

```julia
@inline function _check_bounds(X, i)
    1 <= i <= dim(X) || throw(ArgumentError("there is no index at coordinate $i, since the set is of dimension $(dim(X))"))
end

@inline function center(sih::SymmetricIntervalHull{N}, i::Int) where {N<:Real}
    @boundscheck _check_bounds(S, i)
    return zero(N)
end
```

Reply:
Since this new `_check_bounds` function is shared, we can put it in `Utils/helper_functions.jl`.

Reply:
should I inline every `center(X, i)` function?